### ISSUE TITLE
Provide language selection for TG

### ISSUE THREAD

**Issue by ronja-ui** [teragrep/zep_01#118]
**Description**
To increase Teragrep's accessibility it should be translated to other languages (such as in Finnish) as well.
 
**Use case or motivation behind the feature request**
There has been a request to provide Teragrep software in Finnish language.
 
**Related issues**

 
**Additional context**

**Comment by ronja-ui** [comment]
See related issue in ajs_01 https://github.com/teragrep/ajs_01/issues/244
